asapdiscovery-alchemy/asapdiscovery/alchemy/schema/network.py

```diff
@@ -74,6 +74,28 @@ def get_planning_function(self) -> Callable:
         )
 
 
+class CustomNetworkPlanner(_NetworkPlannerMethod):
+    """Plan a user-set custom network"""
+
+    type: Literal["CustomNetworkPlanner"] = "CustomNetworkPlanner"
+
+    custom_network_file: str = Field(
+        None,
+        description="The path to a CSV file containing edges specified by the user. Each line contains an edge, and ligand names (corresponding to the names in the input SDF) are separated by commas.",
+    )
+
+    def get_planning_function(self) -> Callable:
+        return openfe.ligand_network_planning.generate_network_from_names
+
+    def read_custom_network_csv(csv_path):
```

Comment:
Shouldn't this read the `self. custom_network_file` ? currently this is used as a staticmethod instead. if that is the intention remove the `custom_network_file` attribute and make this a `staticmethod`

Reply:
good catch, made this `staticmethod` instead.